ui-tests/tools/com.liferay.ide.ui.workspace/com.liferay.ide.ui.workspace.wizard/com.liferay.ide.ui.workspace.wizard.validation.tomcat.70/src/com/liferay/ide/ui/workspace/tests/ValidationLiferayWorkspaceTests.java

```diff
@@ -81,12 +95,31 @@ public void checkExsitingLiferayWorkspace() {
 		viewAction.project.closeAndDelete(projectName);
 	}
 
+	@Test
+	public void checkIndexSources() {
+		wizardAction.openNewLiferayWorkspaceWizard();
+
+		wizardAction.newLiferayWorkspace.prepareGradleWithIndexSources(project.getName());
+
+		wizardAction.finish();
+
+		jobAction.waitForNoRunningProjectBuildingJobs();
+
+		viewAction.project.openFile(project.getName(), "gradle.properties");
+
+		validationAction.assertContains("target.platform.index.sources = true", editorAction.getContent());
+
+		viewAction.project.closeAndDelete(project.getName());
+	}
+
 	@Test
 	public void checkInitialState() {
 		wizardAction.openNewLiferayWorkspaceWizard();
 
 		validationAction.assertEquals(PLEASE_ENTER_A_PROJECT_NAME, wizardAction.getValidationMsg(2));
 
+		validationAction.assertTextEquals("7.2", wizardAction.newLiferayWorkspace.liferayVersion());
+
 		validationAction.assertTextEquals(StringPool.BLANK, wizardAction.newLiferayWorkspace.projectName());
 
 		validationAction.assertCheckedTrue(wizardAction.newLiferayWorkspace.useDefaultLocation());
```

Comment:
also add assert for 
1. enable target platform
2. target platform default
3. index source default